internal/storage/bucket_handle.go

```diff
@@ -31,20 +31,20 @@ import (
 	"google.golang.org/api/googleapi"
 )
 
-type bucketHandle struct {
+type BucketHandle struct {
 	gcs.Bucket
-	bucket *storage.BucketHandle
+	BucketObj *storage.BucketHandle
```

Comment:
Why are these changes required. Atleast i dont see them in this PR.

Reply:
fixed